smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/waiters/Variable.kt

```diff
@@ -0,0 +1,35 @@
+package software.amazon.smithy.swift.codegen.waiters
+
+import software.amazon.smithy.codegen.core.SymbolProvider
+import software.amazon.smithy.model.shapes.Shape
+
+// Holds the name of a variable defined in Swift, plus the optionality and the Smithy
+// shape equivalent of its type.
+class Variable(
```

Comment:
Can we move this file out of a the waiters folder and to somewhere more suitable for general utility objects?

Reply:
Moved to `software.amazon.smithy.swift.codegen.utils`

Reply:
Can we rename it then to something domain specific like `JMESPathVariable`?

Reply:
Renamed.